.github/workflows/deploy-to-azure.yml

```diff
@@ -0,0 +1,59 @@
+name: DEPLOY-TO-AZURE
+
+on:
+  workflow_dispatch:
+    permissions:
+      teams:
+        - 'reactor/team'
+
+permissions: read-all
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      SERVICE_NAME: ${{ secrets.AZURE_SERVICE_NAME }}
+      APP_NAME: ${{ secrets.AZURE_APP_NAME }}
+      JAR_PATH: ${{ github.workspace }}/build/libs/projectreactor-home.jar
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+      - uses: actions/setup-java@9704b39bf258b59bc04b50fa2dd55e9ed76b47a8 # v4
+        with:
+          distribution: 'temurin'
+          java-version: 8
+      - uses: gradle/gradle-build-action@29c0906b64b8fc82467890bfb7a0a7ef34bda89e # v3
+        name: build
```

Comment:
maybe the named steps should begin with name and then specify what action they use

Reply:
I have corrected this, see https://github.com/reactor/projectreactor.io/pull/147/commits/a427a6d46d85ee21b003df77a9efb7207e20b1f5